upup/pkg/fi/cloudup/containerd.go

```diff
@@ -168,22 +168,26 @@ func findContainerdVersionHash(arch architectures.Architecture, version string)
 
 func findAllContainerdHashesAmd64() map[string]string {
 	hashes := map[string]string{
-		"1.3.4": "4616971c3ad21c24f2f2320fa1c085577a91032a068dd56a41c7c4b71a458087",
-		"1.3.9": "96663699e0f888fbf232ae6629a367aa7421f6b95044e7ee5d4d4e02841fac75",
-		"1.4.0": "b379f29417efd583f77e095173d4d0bd6bb001f0081b2a63d152ee7aef653ce1",
-		"1.4.1": "757efb93a4f3161efc447a943317503d8a7ded5cb4cc0cba3f3318d7ce1542ed",
-		"1.4.2": "9d0fd5f4d2bc58b345728432b7daac75fc99c1da91afa4f41e6103f618e74012",
-		"1.4.3": "2697a342e3477c211ab48313e259fd7e32ad1f5ded19320e6a559f50a82bff3d",
+		"1.3.4":  "4616971c3ad21c24f2f2320fa1c085577a91032a068dd56a41c7c4b71a458087",
+		"1.3.9":  "96663699e0f888fbf232ae6629a367aa7421f6b95044e7ee5d4d4e02841fac75",
+		"1.3.10": "69e23e49cdf1232d475a77bf7ecd7145ff4a80295154e190125c4d8a20e241da",
+		"1.4.0":  "b379f29417efd583f77e095173d4d0bd6bb001f0081b2a63d152ee7aef653ce1",
+		"1.4.1":  "757efb93a4f3161efc447a943317503d8a7ded5cb4cc0cba3f3318d7ce1542ed",
+		"1.4.2":  "9d0fd5f4d2bc58b345728432b7daac75fc99c1da91afa4f41e6103f618e74012",
+		"1.4.3":  "2697a342e3477c211ab48313e259fd7e32ad1f5ded19320e6a559f50a82bff3d",
+		"1.4.4":  "96641849cb78a0a119223a427dfdc1ade88412ef791a14193212c8c8e29d447b",
 	}
 
 	return hashes
 }
 
 func findAllContainerdDockerMappings() map[string]string {
 	versions := map[string]string{
-		"1.3.7": "19.03.13",
-		"1.3.9": "19.03.14",
-		"1.4.3": "20.10.0",
+		"1.3.7":  "19.03.13",
+		"1.3.9":  "19.03.14",
+		"1.3.10": "19.03.14",
+		"1.4.3":  "20.10.0",
+		"1.4.4":  "20.10.0",
```

Comment:
I don't think this is correct. There is no v1.4.4 released as part of the docker packages.

Reply:
After reviewing the code, my understanding of the `findAllContainerdDockerMappings` function to install the corresponding version of Docker where the version of containerd doesn't exist, such as for ARM64 architectures.

Reply:
Yes, but only if the desired version exists as part of a Docker package.
If not, it should use the `containerdFallbackVersion`, which should be used in case the selected version is not available for ARM64.

Reply:
I've removed v1.4.4 from the `findAllContainerdDockerMappings` function and re-running the E2E tests now.